Makefile

```diff
@@ -1,7 +1,7 @@
 # assignments
 ASSIGNMENT ?= ""
 IGNOREDIRS := "^(\.git|bin|node_modules)$$"
-ASSIGNMENTS = $(shell find . -maxdepth 1 -mindepth 1 -type d -exec basename -a {} + | sort | grep -Ev $(IGNOREDIRS))
+ASSIGNMENTS = $(shell find . -maxdepth 1 -mindepth 1 -type d | cut -d'/' -f2 | sort | grep -Ev $(IGNOREDIRS))
```

Comment:
The `-exec basename -a` part was introduce by @wilmoore but it failed on Travis CI. So I decided to copy the line from the [xjavascript Makefile, line 4](https://github.com/exercism/xjavascript/blob/master/Makefile). So far, it works, but there is only one assignment. I hope it will continue working with more assignments.

If there is a better option, I don't mind changing it.